gateway/tx.go

```diff
@@ -4,12 +4,16 @@ import (
 	"net/http"
 
 	"github.com/celer-network/goutils/log"
+	"github.com/celer-network/sgn/mainchain"
+	"github.com/celer-network/sgn/proto/chain"
 	"github.com/celer-network/sgn/transactor"
 	"github.com/celer-network/sgn/x/subscribe"
 	"github.com/celer-network/sgn/x/sync"
 	"github.com/celer-network/sgn/x/validator"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/gogo/protobuf/proto"
```

Comment:
let's be consistent and use github.com/golang/protobuf/proto for now in all places